app/frontend/src/utils/timeAgo.ts

```diff
@@ -0,0 +1,8 @@
+import moment from "moment";
+
+export function timeAgo(milliseconds: number) {
+  if (new Date().getTime() - milliseconds < 15 * 60 * 1000) {
+    return "less than 15 minutes ago";
```

Comment:
What if the date is in the future? Or shouldn't this function then be used?

Reply:
I also committed a function for this btw.

Reply:
I should have left a comment, the reason for this function is the backend fuzzes the last active time to 15 minute intervals.